src/countdown.js

```diff
@@ -124,10 +124,10 @@
       }
     }
     // Fix #187
-    if(Math.abs(count) > 1) {
-      return plural;
-    } else {
+    if(Math.abs(count) === 1) {
```

Comment:
Pretty sure this should be simply `if(count !== 1) {`:
* 2 second**S**
* 1 second
* 0 second**S**
* -1 second**S**

It's literally singular only when the count is `1`.

English is odd 😋